File: class19/python/AP1910-RepitaAteSenha.py
# RepitaAteSenha — Aula 19
# Loop repita...até para validar senha

senha_correta = "1234"
while True:
    senha = input("Digite a senha: ")
    if senha == senha_correta:
        print("Acesso permitido!")
        break
    print("Senha incorreta. Tente novamente.")

File: class19/cpp/AP1910-RepitaAteSenha.cpp
#include <iostream>
#include <string>
int main() {
    std::string senhaCorreta = "1234", senha;
    do {
        std::cout << "Digite a senha: "; std::cin >> senha;
        if (senha != senhaCorreta)
            std::cout << "Senha incorreta. Tente novamente." << std::endl;
    } while (senha != senhaCorreta);
    std::cout << "Acesso permitido!" << std::endl;
    return 0;
}